ios/MullvadVPN/WireguardPrivateKey.swift

```diff
@@ -6,7 +6,6 @@
 //  Copyright © 2019 Mullvad VPN AB. All rights reserved.
 //
 
-import CryptoKit
```

Comment:
CryptoKit is iOS 13 only and since we're rolling out iOS 12 support, I had to switch back to Jason's implementation in C.